src/stress/adapter/device_allocation.ts

```diff
@@ -0,0 +1,28 @@
+export const description = `
+Stress tests for GPUAdapter.requestDevice.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('concurrent_device_allocation')
```

Comment:
We might want to pick a word other than "concurrent"? because at some point we will also need to write tests which create objects from different threads (workers), both on separate devices (which can be implemented now, actually) and on shared devices (which we don't support yet).

I don't know what, though. I think it's "concurrent" is also OK.

Reply:
Makes sense. Went with "coexisting" to avoid the ambiguous connotation of "concurrent"